kernel-headers/rdma/mlx5_user_ioctl_cmds.h

```diff
@@ -41,6 +41,25 @@ enum mlx5_ib_create_flow_action_attrs {
 	MLX5_IB_ATTR_CREATE_FLOW_ACTION_FLAGS = (1U << UVERBS_ID_NS_SHIFT),
 };
 
+enum mlx5_ib_dm_methods {
```

Comment:
Make sure this is regenerated, this shouldn't have the dm stuff..

Reply:
Right, done